src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp

```diff
@@ -85,18 +85,20 @@ AieTracePluginUnified::~AieTracePluginUnified() {
   AieTracePluginUnified::live = false;
 }
 
-uint64_t AieTracePluginUnified::getDeviceIDFromHandle(void *handle) {
+uint64_t AieTracePluginUnified::getDeviceIDFromHandle(void *handle, bool hw_context_flow) {
   auto itr = handleToAIEData.find(handle);
 
   if (itr != handleToAIEData.end())
     return itr->second.deviceID;
 
 #ifdef XDP_CLIENT_BUILD
+  (void)(hw_context_flow);
   return db->addDevice("win_sysfspath");
-#elif XDP_VE2_BUILD
-  return db->addDevice("ve2_device");
 #else
-  return db->addDevice(util::getDebugIpLayoutPath(handle)); // Get the unique device Id
+  if(hw_context_flow)
+    return db->addDevice("ve2_device");
```

Comment:
This change needs to be revisited during development of MultiPartition support. Same issue exists in AIE Profile Plugin too.